artifacts/centosstream/centos-stream.go

```diff
@@ -91,7 +91,7 @@ func (c *centos) Inspect() (*api.ArtifactDetails, error) {
 
 	if checksum, exists := checksums[candidate]; exists {
 		return &api.ArtifactDetails{
-			SHA256Sum:            checksum,
+			SHASum:               checksum,
```

Comment:
Rename to `Checksum`?

Reply:
IMHO, we are currently just supporting SHA digests. Therefore, it looks appropriated as it is. Maybe it can be changed as `SHAChecksum`. WDYT?

Reply:
Done.